sdk/src/main/java/com/kin/ecosystem/web/EcosystemWebView.java

```diff
@@ -83,6 +84,27 @@ public void run() {
         }
     }
 
+    public void setTheme(final String kinTheme) {
+        if (Looper.myLooper() != Looper.getMainLooper()) {
+            mainThreadHandler.post(new Runnable() {
+                @Override
+                public void run() {
+                    setTheme(kinTheme);
+                }
+            });
+
+            return;
+        }
+
+        final StringBuilder js = new StringBuilder("kin.setTheme(\"").append(kinTheme).append("\")");
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+            evaluateJavascript(js.toString(), null);
```

Comment:
a potential security risk, make sure input is safe (either by making sure the code path is safe or even more paranoidly by sanitizing the input from js input)

Reply:
will do on another task